cmd/kops/replace.go

```diff
@@ -123,16 +132,32 @@ func RunReplace(f *util.Factory, cmd *cobra.Command, out io.Writer, c *ReplaceOp
 				if cluster == nil {
 					return fmt.Errorf("cluster %q not found", clusterName)
 				}
-				_, err = clientset.InstanceGroupsFor(cluster).Update(v)
+				// @check if the instancegroup exists already
```

Comment:
Just a annoying little personal quirk of mine .. I'm sure i've used it in other PRs here, i'll update and change them :-)